packages/lesswrong/components/sunshineDashboard/SunshineNewPostsList.jsx

```diff
@@ -43,6 +43,8 @@ const withListOptions = {
   collection: Posts,
   queryName: 'sunshineNewPostsListQuery',
   fragmentName: 'PostsList',
+  enableCache: true,
+  fetchPolicy: 'cache-and-network'
```

Comment:
Does this change whether the Sunshine sidebar contents get refetched on navigation?

Reply:
It should still get refetched, but will display content before its refetched from cache